commands.sql

```diff
@@ -0,0 +1,2 @@
+
+CREATE TABLE Student(orm_id INT(255), studentNameVARCHAR(255), studentYearVARCHAR(255), studentGpaINT(255))
```

Comment:
Why isn't this git ignored?

Reply:
No, idea i think my commits may have been behind the original makefile.